ui/ducks/bridge/selectors.ts

```diff
@@ -52,17 +53,53 @@ export const getFromChains = createDeepEqualSelector(
       ),
     ),
 );
+
+export const getFromChain = createDeepEqualSelector(
+  getNetworkConfigurationsByChainId,
+  getProviderConfig,
+  (
+    networkConfigurationsByChainId,
+    providerConfig,
+  ): NetworkConfiguration | undefined =>
+    providerConfig?.chainId
+      ? networkConfigurationsByChainId[providerConfig.chainId]
+      : undefined,
+);
+
 export const getToChains = createDeepEqualSelector(
+  getFromChain,
   getAllBridgeableNetworks,
   (state: BridgeAppState) => state.metamask.bridgeState?.bridgeFeatureFlags,
-  (allBridgeableNetworks, bridgeFeatureFlags) =>
-    allBridgeableNetworks.filter(({ chainId }) =>
-      bridgeFeatureFlags[BridgeFeatureFlagsKey.NETWORK_DEST_ALLOWLIST].includes(
-        chainId,
-      ),
+  (
+    fromChain,
+    allBridgeableNetworks,
+    bridgeFeatureFlags,
+  ): NetworkConfiguration[] =>
+    allBridgeableNetworks.filter(
+      ({ chainId }) =>
+        fromChain?.chainId &&
```

Comment:
Would we want to support the case where a user can select a destination chain before selecting an origin chain?

Reply:
The src chain defaults to the wallet's active chain so this isn't possible at the moment